fusor-ember-cli/app/styles/openstack.scss

```diff
@@ -288,6 +286,10 @@
   padding: 0;
   display: block;
 }
+.deployment-roles-unassigned .draggable-object {
+  width: 150px;
+  display: inline-block;
+}
```

Comment:
@eriknelson, it seems this is the major change, the rest of the files besides package.json seem to just be cleaning up formatting and style. Is this right?